### ISSUE TITLE
linkable edit of public github csv, to pull request

### ISSUE THREAD

**Issue by ajschumacher** [flatsheet/editdata.org#36]
To me this is _the_ killer feature. I want to have a URL I can send to somebody (or a group) like `editdata.org/edit/ajschumacher/somerepo/somefile.csv` where they can make edits or download or whatever and then "save" and submit a pull request.

THIS WOULD BE THE GREATEST THING! :dolphin: :dolphin: :dolphin: 

Is such a feature in the works? Is it already there and I'm missing it?

**Comment by sethvincent** [comment]
You're right, that's needed. It'll take some tinkering to figure out the github api and get the flow right on this, but it's something I'd love to have as well.

**Comment by mattbailey0** [comment]
It's really nice to see you two on an issue to together.  That is all.  <3

**Comment by stvnrlly** [comment]
It's in Ruby rather than Node, but https://github.com/benbalter/redliner provides a similar feature for text files. It might (or might not) be useful for figuring out the API, etc.

**Comment by eshellman** [comment]
Will close issue #38 adding here that another use case is where you have a repo with starter data or a data template, and you don't expect to submit a pull request. Or,  when you want to work on someone else's csv data, and you don't have commit privileges, or, as in my case, you want to try out editdata without giving it read/write access to ALL your private repos.

**Comment by ahdinosaur** [comment]
hey. this would be awesome for [an app](http://holodex.enspiral.com) powered by [a public GitHub repo of csv files](https://github.com/holodex/enspiral-data).
